Makefile

```diff
@@ -10,7 +10,7 @@ install-ui:
 	yarn --cwd ui install --frozen-lockfile
 
 start:
-	docker-compose up --build --force-recreate
```

Comment:
To keep

Reply:
my bad, j'ai cherry pické un commit en trop